demos/py_scripts/gromacs.py

```diff
@@ -8,28 +8,14 @@
 
 FLAGS = flags.FLAGS
 
-flags.DEFINE_string("api_url", "http://localhost:6969",
-                    "Base URL of the Inductiva API.")
-
-flags.DEFINE_string("sim_dir",
-                    None,
-                    "Directory with the simulation inputs.",
-                    required=True)
-
-flags.DEFINE_string("sim_config_filename",
-                    None,
-                    "Name of the input file.",
-                    required=True)
-
-flags.DEFINE_string("protein_filename",
+flags.DEFINE_string("input_dir",
                     None,
```

Comment:
Can you also add the directories of the correct files from the `config_files` please, because when I am testing I am not sure which files I am supposed to use.